Makefile

```
@@ -46,6 +46,10 @@ install: deps-update install-ginkgo
 	@echo "Installing needed dependencies"
 
 unit-tests:
-	UNIT_TEST=true go test ./... -tags=utest
+	UNIT_TEST=true go test ./... -tags=utest -coverprofile=cover.out.tmp
+
+coverage-html: test
+	cat cover.out.tmp >cover.out
+	go tool cover -html cover.out
```

What is a purpose of the tmp file, [see](https://stackoverflow.com/questions/10516662/how-to-measure-test-coverage-in-go)?

```suggestion
	UNIT_TEST=true go test ./... -tags=utest -coverprofile=cover.out

coverage-html: test
	go tool cover -html cover.out
```

---

👍 I removed it.